docs/csharp/misc/cs1021.md

```diff
@@ -1,30 +1,43 @@
 ---
 title: "Compiler Error CS1021"
-ms.date: 07/20/2015
+ms.date: 08/20/2018
 f1_keywords: 
   - "CS1021"
 helpviewer_keywords: 
   - "CS1021"
 ms.assetid: 0346ba58-d7cd-40bd-bcad-b90117fdc9b5
 ---
 # Compiler Error CS1021
+
 Integral constant is too large  
   
- A value assigned to an [integral type](../../csharp/language-reference/keywords/integral-types-table.md) is larger than the largest possible unsigned integer value.  
+A value represented by an integer literal is greater than <xref:System.UInt64.MaxValue?displayProperty=nameWithType>.  
   
- The following sample generates CS1021:  
-  
-```csharp  
+The following sample generates CS1021:  
+
+```csharp
 // CS1021.cs  
-enum F : int  
-{  
-   a=(int)2590000000000000000000   // CS1021  
-}  
-  
-public class clx  
-{  
-   public static void Main()  
-   {  
-   }  
+class Program
+{
+    static void Main(string[] args)
+    {
+        int a = 18_446_744_073_709_552_000;
+    }
 }  
 ```
+
+Note that the following code also generates CS1021:
+
+```csharp
+using System.Numerics;
+
+class Program
+{
+    static void Main(string[] args)
+    {
+        var a = new BigInteger(18_446_744_073_709_552_000);
+    }
+}
+```
+
+For information about how to instantiate a <xref:System.Numerics.BigInteger?displayProperty=nameWithType> instance whose value exceeds the range of the built-in numeric types, see the <xref:System.Numerics.BigInteger?displayProperty=nameWithType> reference page.
```

Comment:
I think that the URL can link to the instantiating section: \<xref:System.Numerics.BigInteger#instantiating-a-biginteger-object?displayProperty=nameWithType>